### PR TITLE
docs(blog): add this week in databend 87

### PR DESCRIPTION
**Pull request by PsiACE**
I hereby agree to the terms of the CLA available at: https://databend.rs/dev/policies/cla/

## Summary

Summary about this PR

Closes #issue

### CODE REVIEW

File: website/blog/2023-04-01-databend-weekly-87.md [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+## Highlights
+
+Here are some noteworthy items recorded here, perhaps you can find something that interests you.
+
+- *[Connecting to Databend with DBeaver](https://databend.rs/blog/dbeaver)*, either by selecting the pre-configured MySQL driver or adding the Databend JDBC driver.
+- Databend provides integration with Redash to help you gain better insights into your data. *[Doc | Integrations - Redash](https://databend.rs/doc/integrations/gui-tool/redash)*
+- Master how to display information of columns in a given table. *[Doc | SHOW COLUMNS](https://databend.rs/doc/sql-commands/show/show-full-columns)*
+- Databend now supports `generate_series` and `range` table functions.
+- Databend now supports the `ai_embedding_vector` function, which returns a 1536-dimensional f32 vectors generated by the OpenAI Embeddings API.
+- Databend added support for `[CREATE | DROP | SHOW] SHARE ENDPOINT` DDL.
```
Comment (PsiACE) [inline website/blog/2023-04-01-databend-weekly-87.md:155]: cc @soyeric128 . Currently lacking relevant documents.

File: website/blog/2023-04-01-databend-weekly-87.md [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+```
+
+Databend supports all aggregate functions as aggregate window functions.
+
+- [Doc | (draft) Aggregate Window Functions](https://databend.rs/doc/sql-functions/window-functions/aggregate-window-functions)
+- [PR #10700 | feat(window): initial impl window function](https://github.com/datafuselabs/databend/pull/10700).
+
+### Suggest Function Name on Typos
+
+Databend has added an intelligent feature that automatically provide the closest matching item when you enter an incorrect function name.
```
Comment (soyeric128) [inline website/blog/2023-04-01-databend-weekly-87.md:67]: provide -> provides

File: website/blog/2023-04-01-databend-weekly-87.md [hunk 3]
```diff
@@ -0,0 +1,181 @@
...
+
+```sql
+#> select base64(1);
+ERROR 1105 (HY000) at line 1: Code: 1008, displayText = error:
+  --> SQL:1:8
+  |
+1 | select base64(1)
+  |        ^^^^^^^^^ no function matches the given name: 'base64', do you mean 'to_base64'?
```
Comment (soyeric128) [inline website/blog/2023-04-01-databend-weekly-87.md:75]: -> No function matches the given name: 'base64'. Did you mean 'to_base64'?

File: website/blog/2023-04-01-databend-weekly-87.md [hunk 4]
```diff
@@ -0,0 +1,181 @@
...
+
+Discover some fascinating code snippets or projects that showcase our work or learning journey.
+
+### Dump Running Async Task Stack
+
+Databend now supports dumping the running async task stacks. Simply visit `http://<admin_api_address>/debug/async_tasks/dump` to capture it in your browser.
+
+![Dump Running Async Task Stack](https://user-images.githubusercontent.com/8087042/228602725-a0440e39-3a65-4939-8826-3b92d381cb39.png)
+
+It is worth noting that this feature does not interfere with the performance of Query during normal operation before accessing the dump address.
```
Comment (soyeric128) [inline website/blog/2023-04-01-databend-weekly-87.md:90]: i dont understand here...

File: website/blog/2023-04-01-databend-weekly-87.md [hunk 5]
```diff
@@ -0,0 +1,181 @@
...
+    }
+```
+
+Calling the `async_backtrace::taskdump_tree` function can obtain information about the asynchronous task tree (collected by `#[async_backtrace::framed]`). Tasks are divided into regular tasks and polling tasks (marked with `[POLLING]`). Record the stack information for each task, and output it to a string sorted by stack depth.
+
+To learn more about how it works, refer to the following link:
+
+- [PR | feat(query): try support dump running async task stack](https://github.com/datafuselabs/databend/pull/10830)
+
+### Meet Databend in Jupyter Notebook
```
Comment (soyeric128) [inline website/blog/2023-04-01-databend-weekly-87.md:117]: ### New Way to Integrate with Jupyter Notebook

File: website/blog/2023-04-01-databend-weekly-87.md (hunk 1 above)
Comment (soyeric128) [reply website/blog/2023-04-01-databend-weekly-87.md:155]: ok

File: website/blog/2023-04-01-databend-weekly-87.md (hunk 5 above)
Comment (soyeric128) [reply website/blog/2023-04-01-databend-weekly-87.md:117]: Is it a driver? if so, use **New Driver to Integrate with Jupyter Notebook**

File: website/blog/2023-04-01-databend-weekly-87.md [hunk 6]
```diff
@@ -0,0 +1,181 @@
...
+- Master how to display information of columns in a given table. *[Doc | SHOW COLUMNS](https://databend.rs/doc/sql-commands/show/show-full-columns)*
+- Databend now supports `generate_series` and `range` table functions.
+- Databend now supports the `ai_embedding_vector` function, which returns a 1536-dimensional f32 vectors generated by the OpenAI Embeddings API.
+- Databend added support for `[CREATE | DROP | SHOW] SHARE ENDPOINT` DDL.
+
+## What's Up Next
+
+We're always open to cutting-edge technologies and innovative ideas. You're more than welcome to join the community and bring them to Databend.
+
+### Collect Sled Metrics
```
Comment (soyeric128) [inline website/blog/2023-04-01-databend-weekly-87.md:161]: I dont understand here, we already use sled to capture metrics?

Comment (PsiACE) [reply website/blog/2023-04-01-databend-weekly-87.md:161]: Not really, we just need to collect metrics from the sled.

File: website/blog/2023-04-01-databend-weekly-87.md (hunk 3 above)
Comment (PsiACE) [reply website/blog/2023-04-01-databend-weekly-87.md:75]: This is an error report in a test, I just copied it.

File: website/blog/2023-04-01-databend-weekly-87.md (hunk 6 above)
Comment (PsiACE) [reply website/blog/2023-04-01-databend-weekly-87.md:161]: **Collect Metrics from Sled**

File: website/blog/2023-04-01-databend-weekly-87.md (hunk 4 above)
Comment (PsiACE) [reply website/blog/2023-04-01-databend-weekly-87.md]: Already deleted to avoid complexity.

**Pull request merged**